client/src/components/AuditAdmin/Setup/Review/Review.tsx

```diff
@@ -305,14 +348,45 @@ const Review: React.FC<IProps> = ({
                 ballots cast
               </p>
             )}
-            {cvrsUploaded && contest.cvrChoiceNameConsistencyError && (
-              <Callout intent="warning" style={{ marginBottom: '10px' }}>
+            {cvrsUploaded &&
+              isContestChoiceNameStandardizationComplete &&
+              contest.cvrChoiceNameConsistencyError && (
                 <CvrChoiceNameConsistencyError
                   error={contest.cvrChoiceNameConsistencyError}
                   jurisdictionNamesById={jurisdictionsById}
                 />
-              </Callout>
-            )}
+              )}
+            <StandardizeContestChoiceNamesCallout
+              contest={contest}
+              disabled={locked}
+              openDialog={() =>
+                setIsContestChoiceNameStandardizationDialogOpen(true)
+              }
+              standardizations={contestChoiceNameStandardizations}
+            />
+            <StandardizeContestChoiceNamesDialog
+              contest={contest}
+              isOpen={isContestChoiceNameStandardizationDialogOpen}
+              jurisdictionsById={jurisdictionsById}
+              onClose={() =>
+                setIsContestChoiceNameStandardizationDialogOpen(false)
+              }
+              standardizations={contestChoiceNameStandardizations}
+              standardizedContestChoiceNames={
+                (standardizedContests ?? []).find(c => c.name === contest.name)
+                  ?.choiceNames ?? []
+              }
+              updateStandardizations={async (
+                newStandardizations: IContestChoiceNameStandardizations
+              ) => {
+                await updateContestChoiceNameStandardizations.mutateAsync(
+                  newStandardizations
+                )
+              }}
+              // Reset the form state within the dialog component any time persisted
+              // standardizations change
+              key={JSON.stringify(contestChoiceNameStandardizations)}
```

Comment:
I wonder if it should just reset every time you open the dialog

Reply:
Good call! https://github.com/votingworks/arlo/pull/1948/commits/7242864424da6dd2cef504ffafeaa50704310372